src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: calc(198px - 32px);
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  border-radius: 5px;
+  border: 1px solid var(--gray-accent);
+  background: var(--white);
+
+  transition: all 400ms ease;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin: 32px 20px 40px;
+  }
+
+  &__name {
+    color: var(--main-accent);
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments). Fix all cases